app/serializers/form_serializer.rb

```diff
@@ -23,15 +23,16 @@ def updated_at
   end
 
   def urls
+    user = defined?(current_user) ? current_user : scope[:user]
```

Comment:
Why not like in the reset of serializers? 

user = scope[:user] || @instance_options[:user]